openlibrary/templates/admin/people/index.html

```diff
@@ -37,6 +38,19 @@ <h2>$_("Find Account")</h2>
             </form>
         </td>
     </tr>
+    <tr>
+        <td>$_("IA ID:")</td>
+        <td>
+            <form method="GET" name="form-ia-id">
+                <input type="text" name="ia_id" id="ia-id" value="$ia_id">
```

Comment:
Apparently this must begin with `@`, so make that clearer.

```suggestion
                <input type="text" name="ia_id" id="ia-id" pattern="^@.*$" placeholder="e.g. @foobar">
```